pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "meshtastic"
-version = "2.5.12"
+version = "2.6.0"
```

Comment:
I didn't think to update this and should have, but if you're going to here, set it to `2.6.0a1` or something like that. Then the release automation will increment it properly (to `2.6.0` rather than to `2.6.1`).

Reply:
Heh, failed to include this in the actual review. Well, anyway